packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -114,6 +105,7 @@ export default function SidebarNavigationScreenGlobalStyles() {
 					shouldShowGlobalStylesFooter && (
 						<SidebarNavigationScreenDetailsFooter
 							record={ revisions?.[ 0 ] }
+							recordCount={ revisionsCount }
```

Comment:
While working on https://github.com/WordPress/gutenberg/pull/66606 I was confused by the name of the `record` prop. Actually, to my understanding, it's the record of the _last_ revision. Could this be renamed to `lastRecord` for better clarity?
Similarly, the new prop `recordCount` isn't that clear to me, as it's actually the _revisions count_. Wouldn't be batter to just make it `revisionsCount={ revisionsCount }`?